datajunction-query/.coveragerc

```diff
@@ -2,7 +2,9 @@
 [run]
 branch = True
 source = dj
-# omit = bad_file.py
+omit =
+    djqs/config.py
```

Comment:
Why do we need to omit these files?

Reply:
The logic in here is thoroughly covered since we use `get_settings()` all over the place, but the missing coverage is because we now directly use environment variables in config.py (instead of relying on the pydantic abstraction which does the same under the hood). I was weighing writing a bunch of tests to make sure `os.getenv` works but that seemed silly so I opted to just exclude this settings class and the DJUnknownCatalog exception. Let me know if this makes you nervous and I can write some more tests for these.